src/pages/@owner/food-truck-form/hooks/use-time.ts

```
@@ -0,0 +1,183 @@
+    // 기존 날짜 업데이트
+    const updatedDates = currentDates.map(date =>
+      date.id === id ? { ...date, ...dateData } : date
+    );
+
+    setValue('availableDates', updatedDates, {
+      shouldValidate: true,
+    });
+  };
+
+  const removeAvailableDateById = (
+    event: React.MouseEvent<SVGSVGElement>,
```

ESLint

[error] 'React' is not defined.

(no-undef)